wheel/src/api.rs

```diff
@@ -5,16 +5,34 @@ use clvmr::chia_dialect::ChiaDialect;
 use clvmr::cost::Cost;
 use clvmr::reduction::Response;
 use clvmr::run_program::run_program;
-use clvmr::serde::{node_from_bytes, serialized_length_from_bytes};
+use clvmr::serde::{node_from_bytes, parse_through_clvm_object, serialized_length_from_bytes};
 use clvmr::{LIMIT_HEAP, LIMIT_STACK, MEMPOOL_MODE, NO_UNKNOWN_OPS};
 use pyo3::prelude::*;
 use pyo3::wrap_pyfunction;
 
+struct ReadPyAny<'py>(&'py PyAny);
+
+impl<'py> std::io::Read for ReadPyAny<'py> {
+    fn read(&mut self, b: &mut [u8]) -> std::result::Result<usize, std::io::Error> {
+        let r: Vec<u8> = self.0.call1((b.len(),))?.extract()?;
+        let (p0, _p1) = b.split_at_mut(r.len());
+        p0.copy_from_slice(&r);
+        Ok(r.len())
+    }
+}
+
 #[pyfunction]
 pub fn serialized_length(program: &[u8]) -> PyResult<u64> {
     Ok(serialized_length_from_bytes(program)?)
 }
 
+#[pyfunction]
+pub fn skip_clvm_object(obj: &PyAny) -> PyResult<()> {
```

Comment:
`obj` is a python input stream, right? I think how this function is meant to be used warrants a comment. and perhaps `obj` could have a better name (unless I misunderstand it)